src/components/radioinput.js

```diff
@@ -36,7 +37,8 @@
       RadioGroup,
     } = window.MaterialUI.Core;
     const isDev = env === 'dev';
-    const modelProperty = getProperty(actionProperty.modelProperty || '') || {};
```

Comment:
Why still support the `actionProperty` when that won't be sent to the component anymore since you changed it to `property`?